index.html

```diff
@@ -241,5 +241,41 @@ <h3 class="about-me-item">Skills</h3>
                 </li>
             </ul>
         </section>
+        
+        <footer class="contact-us-footer">
+            <h3 class="contact-us-text">
+                I'm always interested in hearing about 
+                new projects, so if you'd like to chat 
+                please get in touch.
+            </h3>
+
+            <form class="contact-us-form" action="https://formspree.io/f/xknyrjzn" method="POST">
+                <input name="full_name" class="contact-us-input" type="text" placeholder="Full name" maxlength="30" required/>
+                <input name="email" class="contact-us-input" type="email" placeholder="Email" maxlength="30" required/>
+                <textarea name="message" class="contact-us-textarea" placeholder="Enter Text here" maxlength="500"></textarea>
+                <button type="submit" class="see-project-btn">Get in touch</button>
+            </form>
+
+            <ul class="footer-icons social-media-icons" >
+                <li class ="social-icon">
+                    <a href="https://twitter.com"><img src="images/twitter.svg" alt="twitter"></a>
+                </li>
+                <li class ="social-icon">
+                    <a href="https://linkedin.com"><img src="images/Linkedin_icon.svg" alt="LinkedIn"></a>
+                </li>
+                <li class ="social-icon">
```

Comment:
- [ ]  Kindly use space-less equal signs. `<li class="social-icon">` instead of `<li class ="social-icon">`